static/app/views/issueList/searchBar.tsx

```diff
@@ -168,5 +168,9 @@ const RecommendedWrapper = styled('div')`
     border: 1px solid ${p => p.theme.border};
     padding: ${space(1.5)} ${space(2)};
     margin: 0;
+
+    &:not(:first-child):not(.group-child) {
```

Comment:
This works but I think I'd rather lower the specificity of the other declaration than have to override it here. Not sure how difficult that would be though